### ISSUE TITLE
Krub - "i" printing off-center

### ISSUE THREAD

**Issue by aeioddd** [google/fonts#5120]
Hello,

Some PC users on my team have been having an issue with Krub: when the font is printed, the "i"s all print off-center (although they display fine on screen). This happens with PDFs and word docs. Mac users are not having the same issue.

Has anyone else experienced this issue and found a fix?

Thank you!

<img width="940" alt="Krub PC printing issue" src="https://user-images.githubusercontent.com/111927907/186281217-a1452f27-5cec-4462-aa5f-c7223a012f30.png">

**Comment by moyogo** [comment]
Thank you for reporting.

This is a common bug in printer drivers that is triggered by nested components in the font. See for example #2848.
The nested components in this case are the "i" using "uni0307" (combining dot above) as a component, and "uni0307" using "dotaccent" (modifier letter dot above) as a component.
This can be fixed in the sources or when building the fonts from sources, rather than in the printer drivers.

**Comment by kenmcd** [comment]
> and found a fix?

Another work-around is to convert the fonts to OpenType-PS (.otf) - which do not have components. Then the printer cannot mess-up the components (since there are none).

**Comment by aeioddd** [comment]
Thank you both!

For the .otf fix, is that as simple as changing the file name from .tff to .otf? Or should I use a converter like https://convertio.co/ttf-otf/?

**Comment by kenmcd** [comment]
> Thank you both!
> 
> For the .otf fix, is that as simple as changing the file name from .tff to .otf? Or should I use a converter like https://convertio.co/ttf-otf/?

No, you cannot simply change the file extension. You have to actually convert the fonts from OpenType-TT (`.ttf`) which have TrueType curves, to OpenType-PS (`.otf`) which have PostScript curves. So you will need a tool which can do that conversion properly. That conversion will eliminate the use of components inside the fonts.

**Comment by kenmcd** [comment]
@aeioddd 
Here try these (quickie conversion using TransType).
[Krub-OTF-conversion.zip](https://github.com/google/fonts/files/9420303/Krub-OTF-conversion.zip)
